helpers/updateMemberRank.ts

```diff
@@ -65,6 +65,11 @@ export const updateMemberRank = async (
         memberData.accountProgression.quiver = true;
     }
 
+    if ((latestSnapshotData?.activities?.collections_logged.score || 0)  > 0) {
```

Comment:
Add test in the related `updateMemberRank.test.ts` file. There should be enough examples for you there to figure out how it works :)